modules/setting/setting.go

```diff
@@ -1186,6 +1187,7 @@ func newService() {
 	Service.DefaultKeepEmailPrivate = sec.Key("DEFAULT_KEEP_EMAIL_PRIVATE").MustBool()
 	Service.DefaultAllowCreateOrganization = sec.Key("DEFAULT_ALLOW_CREATE_ORGANIZATION").MustBool(true)
 	Service.DefaultEnableTimetracking = sec.Key("DEFAULT_ENABLE_TIMETRACKING").MustBool(true)
+	Service.DefaultEnableDependencies = sec.Key("DEFAULT_ENABLE_DEPENDENCIES").MustBool(true)
```

Comment:
should be false by default

Reply:
@strk I agree with this one.

Reply:
Since there's no migration to enable them on each repo. I think it's fine to keep it enabled for _new_ repositories though 🤔